spce/init.py

```diff
@@ -0,0 +1,12 @@
+import signac
+import sys
+
+if len(sys.argv) != 2:
```

Comment:
This logic is another possible utility function. Albeit the gains for this one would be less.